assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -671,6 +671,7 @@ public Assignment addAssignment(String context) throws PermissionException {
         Assignment assignment = new Assignment();
         assignment.setContext(context);
         assignment.setAuthor(sessionManager.getCurrentSessionUserId());
+        assignment.setPosition(0);
```

Comment:
What does this mean?

Reply:
Every assignment's position starts at "0".